locales/en-US/messages.properties

```diff
@@ -68,6 +68,86 @@ navViewTutorial=TUTORIAL
 navToggleAutoUpdateTitle=Toggle Preview Auto-refresh
 navToggleAutoUpdateLabel=Auto
 
+# Code Snippet Data
+snippetHTMLComment=Comment
+snippetHTMLCommentTitle=Add a Comment
+snippetHTMLCommentData=<!-- Your HTML comment here -->\n
```

Comment:
@TheoChevalier I would love your input here. I'm not sure if this is a good way to localize code snippets that we can insert into Thimble's editor. I wanted to localize code snippets that have text content in them. However, each string has a bunch of `\n`'s in them as well as spaces before tag names for formatting purposes when they are rendered. I don't think I was able to come up with a way around this. Let me know what you think.